packages/nimble-components/src/label-provider/rich-text-editor/name-utils.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Removes the richTextEditor prefix and camelCases the input token name.
+ * (The design token name has the richTextEditor prefix, but the properties on the LabelProviderRichTextEditor do not, as they're already
+ * scoped to the rich-text-editor only)
+ */
+export function removeRichTextEditorPrefixAndCamelCase(
+    jsTokenName: string
+): string {
+    // Example: 'richTextEditorToggleBulletListLabel' => 'toggleBulletListLabel'
+    return jsTokenName.replace(
+        /^richTextEditor(\w)(\w+)/,
```

Comment:
Rather than duplicating this logic from the table, could you pass the prefix in as a parameter and move the utility to a shared location like `label-provider/base/tests`?

Reply:
Ah good catch seeing it's only used in tests 👍

Reply:
Moved to `label-provider/base/tests/label-name-utils.ts` file and reused in 'table' and 'rich-text'